src/index.js

```diff
@@ -29,18 +29,27 @@ export default class ReactPasswordStrength extends React.Component {
     });
   }
 
-  checkValidity(score, length) {
-    const { minScore, minLength } = this.props;
-    return score >= minScore && length >= minLength;
+  tooShort(password) {
+    const { minLength } = this.props;
+    return minLength && password.length < minLength;
```

Comment:
can we rename this to `checkLength` or some more meaningful method name?

and here we can just write `password.length < minLength;` because `minLength` will always be defined